src/prescriptions/prescription-tab-lists.component.tsx

```diff
@@ -122,87 +72,14 @@ const PrescriptionTabLists: React.FC = () => {
             />
           </div>
           <TabPanels>
-            <TabPanel>
```

Comment:
I extracted the entire tab panel out into a new component so that we could reuse it for each of the tabs.